File: CmdMents/God/ClingToGod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.God
{
    class ClingToGod : FearGod
    {
        public ClingToGod()
        {
            base.Book = CommandmentBook.Deuteronomy;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 10;
            base.CommandmentType = CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.Obeyed;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 12;
            base.ShortSummary = "Cling to God and serve him.";
            base.Text = "Fear the LORD your God and serve him. Hold fast to him and take your oaths in his name.";
            base.Verse = 20;
        }
    }
}

File: EtzMitzvot/Commandments/God/clingToGod.ts
///<reference path='../commandmentBase.ts' />

class ClingToGod extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Cling to God and serve him.";
        this.text = "Fear the LORD your God and serve him. Hold fast to him and take your oaths in his name.";
        this.book = TorahBook.Deuteronomy;
        this.chapter = 10;
        this.verse = 20;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.Binding;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 5;

        this.children = [

        ];
    }
}  